.travis.yml

```diff
@@ -1,19 +1,21 @@
+sudo: false
+os:
+  - linux
```

Comment:
why `linux`?
I mean,  I guess I always assumed this was the case,  but,  wouldn't it be default?